wdio.conf.js

```diff
@@ -16,11 +15,11 @@ const config = {
 
   seleniumDocker: {
     enabled: !process.env.TRAVIS,
-    cleanup: true,
+    cleanup: false,
```

Comment:
Could just remove this since default is false

Reply:
probably could.. i have there more for a documentation of default settings